src/screens/FeedScreen/RenderList.js

```diff
@@ -296,7 +309,9 @@ RenderListFeed.propTypes = {
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
   onHeaderOptionClicked: PropTypes.func,
-  onPreviewCommentPress: PropTypes.func
+  onPreviewCommentPress: PropTypes.func,
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props `onPreviewCommentPress`, `onDeletePost`, and `isSelf` are added to PropTypes. This ensures type checking for these props. This is a good practice and improves code robustness.
```diff
+   onPreviewCommentPress: PropTypes.func,
+   onDeletePost: PropTypes.func,
+   isSelf: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->